src/helpers.ts

```diff
@@ -1943,11 +1943,20 @@ Note that in this case, the contract deployment will not behave the same if depl
 
     let data = '0x';
     if (options.execute) {
-      const diamondContract = new Contract(
+      let executeContractABI;
+      if (options.execute.contractAddress) {
+        const artifact = await partialExtension.getExtendedArtifact(
+            options.execute.contractAddress
```

Comment:
getExtendedArtifact expect a name, not an address

Reply:
You are right, maybe we should accept a `Contract` object instead?

Reply:
or a name

Reply:
Right, right.